uber/templates/signups/food_restrictions.html

```diff
@@ -23,16 +23,28 @@
     {% checkgroup fr.standard %}
 
     <br/> <br/>
-    <select name="sandwich_pref">
-        <option value="">What is Your Preferred Sandwich?</option>
-        {% options c.SANDWICH_OPTS fr.sandwich_pref %}
-    </select>
-    {% checkbox fr.no_cheese %} No cheese
+    Please select sandwich options you like. This is to determine how much to buy: <br/>
+    {% checkgroup fr.sandwich_pref %}
 
-    <br/> <br/> Let us know if you have any other allergies or restrictions: <br/>
+    <br/> <br/>
+    Let us know if you have any other allergies or restrictions,
+    please use commas to separate each allergy and food restriction.
+    (We do not serve seafood or use ingredients with seafood in it.)
     <textarea name="freeform" rows="3" cols="80">{{ fr.freeform }}</textarea>
-    
+
     <br/> <br/> <input type="submit" value="Upload Restrictions" />
+
+    <br/> <br/>
+    [1] Does not eat meat. We understand that there are different types of vegetarians, but due to the amounts and restrictions we have to cook, we cannot accommodate all types. If you only eat chicken, please indicate that in the text box.
+    <br/> <br/>
+    [2] Does not eat animal by-products
 </form>
 
```

Comment:
This part is mildly amusing; since "Vegetarian" and "Vegan" are generated from our config, we need to use Javascript to add footnotes to them.